File: 2.ödev/ColonyClash/src/BTaktik.c
/**
*
* @author Sena Nur Erdem     sena.erdem1@ogr.sakarya.edu.tr
* @since 5 Mayıs 2023
*/
#include "BTaktik.h"
#include <stdlib.h>

int btaktik_savas(Taktik *taktik) {
    int rastgele_sayi = (rand() % 500) * 2 + 1; // Rastgele bir tek sayı üret, 1 ile 999 arasında
    return rastgele_sayi;
}

Taktik *btaktik_olustur() {
    return taktik_olustur(btaktik_savas);
}

File: 3.ödev/Colony/src/colony/BTaktik.java
/**
*
* @author Sena Nur Erdem     sena.erdem1@ogr.sakarya.edu.tr
* @since 23 Mayıs 2023
* <p>
*  Rastgele bir tek sayı üreten bir taktik sınıfı.
* </p>
*/


package colony;
import java.util.Random;

class BTaktik implements Taktik {
    public int savas() {
        Random rand = new Random();
        int rastgele_sayi = (rand.nextInt(500) * 2 + 1); // Rastgele bir tek sayı üret, 1 ile 999 arasında
        return rastgele_sayi;
    }
}